virttest/guest_agent.py

```diff
@@ -552,6 +552,50 @@ def set_vcpus(self, action):
         return self.cmd(cmd=cmd, args=action)
 
     @error_context.context_aware
+    def get_time(self):
```

Comment:
__has_command is a private function, if some day we want to add some case which base on your test class, it will not callable.